### DISCUSSION TITLE
Ability to add presets to quickly chnage brightness

### DISCUSSION THREAD

**Discussion by superuser-me** [MonitorControl/MonitorControl#1503] [Ideas]
Mostly we adjust _lower brightness at night_ and _higher brightness in daytime_,

It would be nice to have a feature where we can **quickly change brightness presets**.
You can set a preset brightness and a bind a keyboard shortcut with that,

or if you want to really next level then automated preset activation based on time,
so at night it will automatically change brightness.

**Comment by dranovsky** [comment]
I saw the feature https://github.com/MonitorControl/MonitorControl/issues/1106 created by @waydabber got closed by bot last summer. I'd really like this feature too, and happy to contribute some $ to the pot for it.

**Reply by waydabber** [reply]
I'll be adding this feature to [BetterDisplay](https://github.com/waydabber/BetterDisplay#readme).

https://github.com/waydabber/BetterDisplay/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+preset

Note: development of MonitorControl has mostly ceased as the team's decision was that it fulfilled its original vision. Some service/compatibility updates might still come though.
